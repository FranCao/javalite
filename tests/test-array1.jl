int[] intArr = [1,2,3];
int i = intArr[2];
print(i);
string[] strArr = ["h", "e", "y"];
for (i = 0 ; i < 3 ; i = i + 1) {
   print(strArr[i]);
}

