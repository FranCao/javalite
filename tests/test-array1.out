3
h
e
y